costs/costs.go

```diff
@@ -22,8 +22,7 @@ import (
 
 // HandleRequest is a dummy request handler function. It does nothing except
 // some logging and returns static data.
-func HandleRequest(response http.ResponseWriter, request *http.Request) {
-	logger := jsonlog.LoggerFromContextOrDefault(request.Context())
+func HandleRequest(response http.ResponseWriter, request *http.Request, logger jsonlog.Logger) {
 	logger.Debug("Request headers.", request.Header)
```

Comment:
Request handlers should have the following signature :
```go
func Handler(r *http.Request, a routes.Arguments) (int, interface{})
```
Also you should get the logger from the request context using `jsonlog.LoggerFromContextOrDefault`.

Since these are not part of the master repository yet, you should base this on the schubev/trackit repository instead, and rebase.